src/main/java/picard/util/IntervalListToBed.java

```diff
@@ -23,11 +23,11 @@
  */
 package picard.util;
 
-import htsjdk.samtools.util.CollectionUtil;
-import htsjdk.samtools.util.IOUtil;
-import htsjdk.samtools.util.Interval;
-import htsjdk.samtools.util.IntervalList;
-import htsjdk.samtools.util.RuntimeIOException;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.util.*;
```

Comment:
there is a preference for explicit imports...can you ask IntelliJ to use explicit imports?